GALIL/iocBoot/iocGALIL-IOC-01/st-common.cmd

```diff
@@ -77,11 +77,14 @@ motorUtilInit("$(MYPVPREFIX)$(IOCNAME):")
 ##ISIS## Stuff that needs to be done after iocInit is called e.g. sequence programs 
 < $(IOCSTARTUP)/postiocinit.cmd
 
+stringiftest("HASMTRCTRL", "$(MTRCTRL=)", 0, 0)
+$(IFNOTHASMTRCTRL) errlogSev("MTRCTRL has not been set", 2)
+
 # Save motor positions every 5 seconds
-$(IFNOTTESTDEVSIM) create_monitor_set("$(IOCNAME)_positions.req", 5, "P=$(MYPVPREFIX)MOT:,CCP=$(MTRCTRL)")
+$(IFHASMTRCTRL) $(IFNOTSIM) create_monitor_set("$(IOCNAME)_positions.req", 5, "P=$(MYPVPREFIX)MOT:,CCP=$(MTRCTRL)")
 
 # Save motor settings every 30 seconds
-$(IFNOTTESTDEVSIM) create_monitor_set("$(IOCNAME)_settings.req", 30, "P=$(MYPVPREFIX)MOT:,CCP=$(MTRCTRL)")
```

Comment:
If MTRCTRL is not defined, instead of being commented out this line causes 

`macLib: macro MTRCTRL is undefined (expanding string $(IFHASMTRCTRL) $(IFNOTSIM) create_monitor_set("$(IOCNAME)_settings.req", 30, "P=$(MYPVPREFIX)MOT:,CCP=$(MTRCTRL)"))`

This is an explicit step in the IOC review checklist but I realise it might be quite annoying to fix. It it's hard then don't worry about it.

Reply:
Yh, I'm aware. My rationale is that the error message is actually clearer than commenting the line out. In which case I guess you could argue that we could remove the IFHASMTRCTRL macro from those lines but it still guards against MTRCTRL getting set to an empty string and allows future extensions (e.g. being set to an invalid value)